### PR TITLE
Feat/update circuit diagram

### PR DESCRIPTION
**Pull request by MagnusGjosund**
The circuit diagram components will now look more like each other. There's a icon with a popover when hover on every component except the most common components, such as switchboard, circuit, cable and junction box.  

The most common components will have rounded corners, and the rest will have straight corners. 

The electrical component you click on in garden/grid will be highlighted in the diagram. 

When a junction box is missing a cable it spawns a warning icon, and when hovering this icon its says its missing a cable.

### CODE REVIEW

**Comment by MagnusGjosund** [comment]
deploy test

**Comment by MagnusGjosund** [comment]
deploy test

**Comment by MagnusGjosund** [comment]
deploy test

**Comment by MagnusGjosund** [comment]
deploy test

**Comment by MagnusGjosund** [comment]
deploy test

**Comment by MagnusGjosund** [comment]
deploy test

File: libs/shared/src/packages/circuit-diagram/ui/CircuitDiagram.tsx [hunk 1]
```diff
@@ -1,38 +1,47 @@
...
-Icon.add({ heat_trace, cable, junction_box, circuit });
-
-type CircuitRef = Record<string, HTMLDivElement>;
+import {
+  CircuitDiagramSkeleton,
+  Switchboard,
+  HTCable,
+  SpaceHeater,
+  MaybeFirst,
+  Cable,
```
Comment (Gustav-Eikaas) [inline libs/shared/src/packages/circuit-diagram/ui/CircuitDiagram.tsx:17]: Sometimes entities like ControlPanel, Motor, Switch etc are grouped on a global object. like i'e

```TS

import { entities } from 'entities.js'


return <entities.Motor />
```

**Comment by MagnusGjosund** [comment]
deploy test

**Review by Gustav-Eikaas — APPROVED** [review]

**Comment by MagnusGjosund** [comment]
deploy test

**Pull request merged**